recipes/libsndfile/all/conanfile.py

```diff
@@ -16,15 +17,18 @@ class LibsndfileConan(ConanFile):
         "programs": [True, False],
         "experimental": [True, False],
         "with_alsa": [True, False],
-        "with_sqlite": [True, False],
+        "with_sqlite": ["deprecated", True, False],
+        "with_flac_opus_vorbis": [True, False],
     }
     default_options = {
         "shared": False,
         "fPIC": True,
         "programs": True,
         "experimental": False,
         "with_alsa": False,
-        "with_sqlite": True,
+        "with_sqlite": "deprecated",
+        "with_flac_opus_vorbis": True,
```

Comment:
is it possible to have more granular options? as these libraries have difference licenses, probably some consumers will want more granular selection

Reply:
No, `ENABLE_EXTERNAL_LIBS` needs all requirements, or it disables the option.
https://github.com/libsndfile/libsndfile/blob/91128fcd60ab13cca0b1475960c75b07cdca7b04/CMakeLists.txt#L79

Reply:
RFC: I find the name of the option `with_flac_opus_vorbis` a bit weird.
Is it ok?

Reply:
yes, `with_external_libs` sounds better then

Reply:
I have renamed it.